config/ocean/trash_pickup.ini

```diff
@@ -0,0 +1,33 @@
+[base]
+package = ocean
+env_name = puffer_trash_pickup trash_pickup
+policy_name = TrashPickup
+
+[env]
+num_envs = 32768
```

Comment:
This is a lot of envs. You should use 4096 as a starting point, and then set train num envs and workers to 2. This creates 2-fold redundancy async so you can drive env time to 0. Reason for 4096 is that it usually saturates GPU. You don't get benefit most of the time from going higher

Reply:
4096 agents* 
Divide by number of envs if multiagent. Ideally num agents is a power of 2

Reply:
Completed (to be pushed).